frontend/src/auth/__tests__/handleNotAuthorizedErrors.test.ts

```diff
@@ -39,9 +39,15 @@ describe('given the application config', () => {
       it('should redirect to the authorization server', () => {
         handleNotAuthorizedErrors(mockAppConfig)(mockRequestPromise).catch(() => {
             expect(global.window.location.replace).toHaveBeenCalledTimes(1)
-            expect(global.window.location.replace).toHaveBeenCalledWith('some-url?response_type=code&client_id=some-id&scope=some-list&redirect_uri=some-uri')
-          })
+            expect(global.window.location.replace).toHaveBeenCalledWith(
+                expect.stringContaining('some-url?response_type=code&client_id=some-id&scope=some-list&redirect_uri=some-uri')
+            )
+            // 'as any' to avoid breaking npx tsc -p tsconfig.json
```

Comment:
There's no need to comment type assertions :)

Reply:
I'll remove it